tests/test_rows.py

```diff
@@ -80,6 +83,35 @@ def test_no_query_parameter_provided(client):
         client.rows.query()
 
 
+def test_transaction(client):
+    lastName = str(uuid.uuid4())
+    content = {
+        "musician": {
+            "lastName": lastName,
+            "firstName": "DoesntMatter",
+            "dob": "2000-01-01",
+        }
+    }
+    uri = "/temp/musician5.json"
+
+    with client.transactions.create() as tx:
+        query = f'op.fromView("test", "musician")'
+        query = f'{query}.where(op.eq(op.col("lastName"), "{lastName}"))'
+        results = client.rows.query(query, tx=tx)
+        assert len(results) == 0
```

Comment:
This makes me think that if no rows are returned, we should probably return `None` instead of the empty array I chose in the prior PR. I'll submit a follow-up PR for that.

Reply:
I'm not convinced, but I'm happy to defer. Perhaps "None" is more pythonic?

Reply:
What I realized is that returning `[]` is confusing because by default, the rows endpoint returns a JSON object with properties of "columns" and "rows". So I had assertions of "len results == 0", then "len results == 1", and the latter failed because I was forgetting that I get an object back, not an array.

So I think returning `None` is more meaningful - there's "no" content, it's not an array or an object.